tools/templates/legion-component/component.js

```diff
@@ -0,0 +1,35 @@
+module.exports = (componentName) => ({
+    content: `import React from 'react';
+    import { getDebugLogger } from '../../../../Models/Services/Utils';
```

Comment:
are these paths all correct relative to the root of the Legion Components folder?

Reply:
yeah, I tested it for Legion folders